ecommerce/templates/oscar/basket/basket.html

```diff
@@ -43,10 +43,12 @@ <h3>{% trans "Your basket is empty" %}</h3>
                                 <a href="{{ support_url }}">
```

Comment:
Why did not you change this instead of the new start_link variable?

Reply:
I'm not quite sure why, but the other templates where the support email is used, define the start link variable as you see here, with the "safe", to avoid html scaping. 

I don't know how to set the "safe" filter (or if it's necessary) in the "captureas" block so I added the "with" block as is used in other templates. I forgot to remove the previous captureas.

Reply:
Ok, In that case I think that you should remove this captureas block since it is not being used.

Reply:
done :+1: